src/sentry/static/sentry/app/utils/discover/eventView.tsx

```diff
@@ -977,6 +1023,19 @@ class EventView {
     };
   }
 
+  getResultsViewShortUrlTarget(slug: string): {pathname: string; query: Query} {
```

Comment:
Afaict, this is just extracting the URL parameters relating to the Global Selection Header and passing them on. And the list can be found [here](https://github.com/getsentry/sentry/blob/7a6c92700c1222660573766137014059239a4c6e/src/sentry/static/sentry/app/constants/globalSelectionHeader.tsx#L1-L8). Unsure why there is a duplicate constant here.

Reply:
This is minimizing the number of params being splatted into the url from the query list, no? generateQueryStringObject puts more into the url? What constant do you mean?

Reply:
I was previously using the `EXTERNAL_QUERY_STRING_KEYS` defined at the top of this file + adding project and environment, so I think Tony meant that it could be replaced by the global selection header URL_PARAMS (correct me if I mistook)

Reply:
Yeah, I meant that `EXTERNAL_QUERY_STRING_KEYS` looks like a duplicated constant from `URL_PARAM`